src/Setup.php

```diff
@@ -199,7 +199,7 @@ public function bcgov_block_theme_custom_logo( $html ) {
 			return $html;
 		}
 		// Otherwise, inject a default logo markup for use in wp-site-logo blocks.
-		$image = '<img src="' . get_stylesheet_directory_uri() . '/dist/images/bc_gov_logo_transparent.ced9da43.png">';
+		$image = '<img class="bcgov-logo" src="' . get_stylesheet_directory_uri() . '/assets/images/bc_gov_logo_transparent.svg">';
```

Comment:
why are we not using dist folder, this seems like a bad idea.

Reply:
Probably warrants a bigger discussion around what gets built and what is closer to static content ie: patterns and their default assets. I have hit a number of walls with how WP wants to do things in this brave new world of FSE so some of the way things are handled are indicative of throwing my hands in the air after banging my head repeatedly on the keyboard.